cmd/kusk/manifests/api_server.yaml

```diff
@@ -113,7 +122,24 @@ spec:
         - image: kubeshop/kusk-gateway-api:v1.1.10
           imagePullPolicy: Always
           name: kusk-gateway-api
-      serviceAccountName: kusk-gateway-manager
```

Comment:
Why did you remove this?

Reply:
Its the wrong service account. You can see I define the proper service account on https://github.com/kubeshop/kusk-gateway/pull/946/files#diff-070d573c248f6027cc11182e49ba2213f9abcfb3c113da56a41fe91dbf0bd82eR142

Reply:
OK. Cool. Fine.